mono/metadata/cominterop.c

```diff
@@ -3522,10 +3500,10 @@ mono_marshal_safearray_set_value (gpointer safearray, gpointer indices, gpointer
 static void
 mono_marshal_safearray_set_value (gpointer safearray, gpointer indices, gpointer value)
```

Comment:
Isn't there a one-line difference between the WIN32 and non-WIN32 version of this function? As long as you're changing it, maybe pull the ifdef into the body so that the common parts are shared?

Reply:
Good catch.

Reply:
Sort of.